addons/event/models/event_event.py

```diff
@@ -428,24 +430,14 @@ def _compute_event_ticket_ids(self):
         Note that updating event_ticket_ids triggers _compute_start_sale_date
         (start_sale_date computation) so ensure result to avoid cache miss.
         """
-        if self.ids or self._origin.ids:
-            # lines to keep: those with already sent emails or registrations
-            tickets_tokeep_ids = self.env['event.registration'].search(
-                [('event_id', 'in', self.ids or self._origin.ids)]
-            ).event_ticket_id.ids
-        else:
-            tickets_tokeep_ids = []
         for event in self:
             if not event.event_type_id and not event.event_ticket_ids:
                 event.event_ticket_ids = False
                 continue
 
             # lines to keep: those with existing registrations
-            if tickets_tokeep_ids:
-                tickets_toremove = event._origin.event_ticket_ids.filtered(lambda ticket: ticket.id not in tickets_tokeep_ids)
-                command = [(3, ticket.id) for ticket in tickets_toremove]
-            else:
-                command = [(5, 0)]
+            tickets_toremove = event.event_ticket_ids.filtered(lambda ticket: not ticket._origin.registration_ids)
```

Comment:
Same here, could be great to have a test case showing what is fixed, because I am quite confused with the change.